api_tests/search/views/test_views.py

```diff
@@ -0,0 +1,583 @@
+from modularodm import Q
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+from api_tests.utils import create_test_file
+
+from framework.auth.core import Auth
+from framework.mongo import database as database_proxy
+
+from tests.base import ApiTestCase, teardown_database
+from tests.factories import (
+    AuthUserFactory,
+    NodeFactory,
+    ProjectFactory,
+)
+from tests.utils import mock_archive
+
+from website.models import MetaSchema
+from website.project.model import ensure_schemas
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+from website.search import search
+
+
+class ApiSearchTestCase(ApiTestCase):
+
+    def setUp(self):
+        super(ApiSearchTestCase, self).setUp()
+
+        # actually tear down the database and delete elastic indices for each test
+        teardown_database(database=database_proxy._get_current_object())
```

Comment:
Actually, this is no longer necessary since I'm not creating users with specific usernames. And since I'm clearing out the ES index for each test, it doesn't matter that the database isn't being torn down.